src/Fork.php

```diff
@@ -66,10 +66,10 @@ public function run(callable ...$callables): array
             $tasks[] = Task::fromCallable($callable, $order);
         }
 
-        return $this->waitFor(...$tasks);
+        return $this->runTasks(...$tasks);
```

Comment:
It seems that these changes are not required to make the feature work. Make sure that there are only related changes in this PR.